### PR TITLE
[new release] junit_alcotest, junit and junit_ounit (2.0.2)

### PR DESCRIPTION
**Pull request by Khady**
JUnit XML reports generation for alcotest tests

- Project page: <a href="https://github.com/Khady/ocaml-junit">https://github.com/Khady/ocaml-junit</a>
- Documentation: <a href="https://khady.github.io/ocaml-junit/">https://khady.github.io/ocaml-junit/</a>

### CODE REVIEW

File: packages/junit/junit.2.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+maintainer: "Louis Roché <louis@louisroche.net>"
+authors: "Louis Roché <louis@louisroche.net>"
+homepage: "https://github.com/Khady/ocaml-junit"
+bug-reports: "https://github.com/Khady/ocaml-junit/issues"
+license: "LGPLv3+ with OCaml linking exception"
+dev-repo: "git+https://github.com/Khady/ocaml-junit.git"
+doc: "https://khady.github.io/ocaml-junit/"
+tags: ["junit" "jenkins"]
+depends: [
+  "dune" {build & >= "1.0"}
```
Comment (kit-ty-kate) [inline packages/junit/junit.2.0.2/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/junit_alcotest/junit_alcotest.2.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+maintainer: "Louis Roché <louis@louisroche.net>"
+authors: ["Louis Roché <louis@louisroche.net>"]
+homepage: "https://github.com/Khady/ocaml-junit"
+bug-reports: "https://github.com/Khady/ocaml-junit/issues"
+license: "LGPLv3+ with OCaml linking exception"
+dev-repo: "git+https://github.com/Khady/ocaml-junit.git"
+doc: "https://khady.github.io/ocaml-junit/"
+tags: ["junit" "jenkins" "alcotest"]
+depends: [
+  "dune" {build & >= "1.0"}
```
Comment (kit-ty-kate) [inline packages/junit_alcotest/junit_alcotest.2.0.2/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/junit_ounit/junit_ounit.2.0.2/opam [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+maintainer: "Louis Roché <louis@louisroche.net>"
+authors: ["Louis Roché <louis@louisroche.net>" "Simon Cruanes <simon.cruanes.2007@m4x.org>"]
+homepage: "https://github.com/Khady/ocaml-junit"
+bug-reports: "https://github.com/Khady/ocaml-junit/issues"
+license: "LGPLv3+ with OCaml linking exception"
+dev-repo: "git+https://github.com/Khady/ocaml-junit.git"
+doc: "https://khady.github.io/ocaml-junit/"
+tags: ["junit" "jenkins" "ounit"]
+depends: [
+  "dune" {build & >= "1.0"}
```
Comment (kit-ty-kate) [inline packages/junit_ounit/junit_ounit.2.0.2/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/junit_alcotest/junit_alcotest.2.0.2/opam [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "odoc" {with-doc & >= "1.1.1"}
+  "alcotest"
+  "junit"
```
Comment (kit-ty-kate) [inline packages/junit_alcotest/junit_alcotest.2.0.2/opam]:
```suggestion
  "junit" {= version}
```
Doesn't this need a constraint?

File: packages/junit_ounit/junit_ounit.2.0.2/opam [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "odoc" {with-doc & >= "1.1.1"}
+  "ounit"
+  "junit"
```
Comment (kit-ty-kate) [inline packages/junit_ounit/junit_ounit.2.0.2/opam]:
```suggestion
  "junit" {= version}
```
Doesn't this need a constraint?

**Comment by kit-ty-kate** [comment]
Could you return the fixes upstream? Thanks

**Comment by Khady** [comment]
Sorry I missed your messages at first and then got an injury. Thanks for taking care of that. I will import your changes upstream.

**Pull request merged**